docs/api/request-events.md

```diff
@@ -0,0 +1,21 @@
+### restifyDone
+
+After request has been fully serviced, an `restifyDone` event is fired.
+restify considers a request to be fully serviced when either:
+
+1) The handler chain for a route has been fully completed
+2) An error was returned to `next()`, and the corresponding error events have
+   been fired for that error type
+
+The signature is for the `restifyDone` event is as follows:
+
+```js
+function(route, error) { }
```

Comment:
Any reason not to match the API of server after?

Reply:
I just thought it would look strange that we pass req in a req event...